public/apps/account/account-app.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ *   Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { CoreStart } from '../../../../../src/core/public';
+import { AccountNavButton } from './account-nav-button';
+import { fetchAccountInfoSafe } from './utils';
+
+export async function setupTopNavButton(coreStart: CoreStart) {
+  const accountInfo = (await fetchAccountInfoSafe(coreStart))?.data;
+  if (accountInfo) {
+    coreStart.chrome.navControls.registerRight({
+      order: 2000,
```

Comment:
Curious how this number is set.

Reply:
Because the kibana predefined plugin newsfeed (the envelope icon on the screenshot)  is 1000.

Reply:
Add some description